### DISCUSSION TITLE
Option to customize the setup of basic cluster

### DISCUSSION THREAD

**Discussion by rahulrai-in** [Azure/vscode-aks-tools#418] [Ideas]
**Is your feature request related to a problem? Please describe.**
Currently, the extension creates its own version of a basic cluster. I would like to customize the behavior of the command to define a basic cluster.
Also, the name of the basic cluster should follow a standard so that I can identify the test clusters in the portal or extension without having to explore their workloads.

**Mention what platform you want to support the new feature**
VSCode

**Describe the solution you'd like**
The cluster setup parameters and the naming pattern should be read from a configuration file, such as a dotfile, for ease of use.

**Describe alternatives you've considered**
Use the CLI to create a cluster. Commit the script to create a cluster in the repository to share with other developers.

**Additional context**
1. Developers can create their own clusters and not delete them afterward.
2. Without a naming convention, it is hard to identify the clusters that can be safely deleted.

**Comment by rahulrai-in** [comment]
Tagging @nellysattari

**Comment by Tatsinnit** [comment]
Supporting improvements happening and will be part of next release - https://github.com/Azure/vscode-aks-tools/pull/241

**Comment by Tatsinnit** [comment]
☕️💡 There is whole lot of cluster create effort which will happen under #294 - I think it will be worth to keep this issue in mind as fyi as well, @hsubramanianaks and cc: @peterbom

**Comment by Tatsinnit** [comment]
☕️🙏🎊 Given we have delivered an improved UI and other improvements via #241 & #294, I recently went through various discussions realised that we should confer this to open discussion now. 

I feel we should **avoid** creating replication of portal experience so what we have at this stage is nice to have from vscode experience, I am also not want this to be closed either in case we have enough community support its worth to scope this.
